packages/odyssey-react-mui/src/labs/DataView/CardLayoutContent.tsx

```diff
@@ -228,13 +228,10 @@ const CardLayoutContent = <TData extends MRT_RowData>({
                       (cardLayoutOptions.rowActionMenuItems ||
                         hasRowReordering) && (
                         <RowActions
-                          row={row as unknown as MRT_Row<MRT_RowData>} // TODO: FIX THIS! The types are wrong. `row` is incorrectly set in `RowActions` to not be TData.
+                          row={row}
                           rowIndex={currentIndex}
                           rowActionMenuItems={
-                            // TODO: FIX THIS! The types are wrong. `row` is incorrectly set in `RowActions` to not be TData.
-                            cardLayoutOptions.rowActionMenuItems as unknown as (
-                              row: MRT_Row<MRT_RowData>,
-                            ) => any // eslint-disable-line @typescript-eslint/no-explicit-any
+                            cardLayoutOptions.rowActionMenuItems as CardLayoutProps<MRT_RowData>["rowActionMenuItems"]
```

Comment:
What's `CardLayoutProps` that we need an `as` here? Shouldn't this be `TData` and not `MRT_RowData`?

Or even `MRT_RowData<TData>`?